packages/emotion-primitives/src/convertToRNStyles.js

```diff
@@ -0,0 +1,109 @@
+import { StyleSheet } from 'react-primitives'
+import transform from 'css-to-react-native'
+
+// Copied and edited from @emotion/css
+// Cannot use css from @emotion/css directly because of unnecessary checks, __emotion_styles, and registering interpolations in cache.
+function handleInterpolation(interpolation, couldBeSelectorInterpolation) {
+  if (interpolation == null) {
+    return ''
+  }
+
+  switch (typeof interpolation) {
+    case 'boolean':
+      return ''
+    case 'function':
+      return handleInterpolation.call(
+        this,
+        this === undefined ? interpolation() : interpolation(this.props),
+        couldBeSelectorInterpolation
+      )
+
+    default:
+      return interpolation
+  }
+}
+
+// Return evaluated css string
+function createStyles(strings, ...interpolations) {
+  let stringMode = true
+  let styles = ''
+
+  if (strings == null || strings.raw === undefined) {
+    stringMode = false
+    styles += handleInterpolation.call(this, strings, false)
+  } else {
+    styles += strings[0]
+  }
+
+  interpolations.forEach(function(interpolation, i) {
+    styles += handleInterpolation.call(
+      this,
+      interpolation,
+      styles.charCodeAt(styles.length - 1) === 46
+    )
+
+    if (stringMode === true && strings[i + 1] !== undefined) {
+      styles += strings[i + 1]
+    }
+  }, this)
```

Comment:
wouldnt arrow function be good enough here? passing `this` like that is a little bit surprising for the reader at the first moment

Reply:
It was to me too when I first referenced this with @emotion/css. We can change this, no issues.

Reply:
I did this to save a few bytes so that Babel didn’t need to add `var _this = this`. It won’t make a big difference but it helps. I’m fine with changing it though

Reply:
if that's the case I'm fine with keeping it here too